hu.irl.cameractrls.yml

```diff
@@ -0,0 +1,50 @@
+app-id: hu.irl.cameractrls
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+finish-args:
+  - --device=all
+  - --device=dri
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+command: cameractrlsgtk.py
+cleanup:
+  - /include
+  - /lib/cmake
+  - /lib/pkgconfig
+  - '*.a'
+  - '*.la'
+modules:
+  - name: cameractrls
+    buildsystem: simple
+
+    build-commands:
+      - install -D -t /app/bin cameractrls.py cameractrlsgtk.py cameraview.py
+      - install -D -t /app/bin/images images/icon_256.png
+      - install -D -t /app/share/metainfo hu.irl.cameractrls.metainfo.xml
+      - install -D -T images/icon_256.png /app/share/icons/hicolor/256x256/apps/hu.irl.cameractrls.png
+      - desktop-file-edit
+        --set-icon=hu.irl.cameractrls
+        --set-key=Exec --set-value=cameractrlsgtk.py
+        --remove-key=Path cameractrls.desktop
+      - install -D -T cameractrls.desktop /app/share/applications/hu.irl.cameractrls.desktop
+    post-install:
+      - python3 -m compileall /app/bin
+    sources:
+    - type: dir
+      path: .
```

Comment:
```suggestion
```

Reply:
Without this, the  `install -D -t /app/share/metainfo hu.irl.cameractrls.metainfo.xml` doesn't work